src/app/modules/domain/unit/unit.service.ts

```diff
@@ -52,4 +52,13 @@ export class UnitService {
     instructionalEvents = instructionalEvents.map(ie => this.learningObjectMapper.convert(ie));
     return instructionalEvents;
   }
+
+  launchSession(kcId: number) : Observable<unknown> {
+    return this.http.post(environment.apiHost + 'units/knowledge-components/' + kcId + '/session/launch', null);
+  }
+
+  terminateSession(kcId: number) : void {
+    this.http.post(environment.apiHost + 'units/knowledge-components/' + kcId + '/session/terminate', null)
+      .subscribe(() => { });
```

Comment:
For consistency you might want to call subscribe in the component. Also, you don't need an empty lambda in subscribe, you can call it without passing any parameters.

Reply:
Moved subscribe call in https://github.com/Clean-CaDET/platform-tutor-ui-web/pull/27/commits/980ff3832759e35efe83727fad8d5b3bc9a9fcec.